resources/windows/windows_testing.py

```diff
@@ -463,27 +457,24 @@ def build_code_using_visual_studio(self,
             self.set_return_code(1)
             test_run.results[solution_type + " build"] = "Fail"
             return False
+
+        cmd = (
+            'call {} {} && '
+            'msbuild /nodeReuse:false /t:Rebuild /p:Configuration={},Platform={}, PlatformToolset={} /m "{}"'
```

Comment:
There's an extra space compared to what there was before.
```suggestion
            'msbuild /nodeReuse:false /t:Rebuild /p:Configuration={},Platform={},PlatformToolset={} /m "{}"'
```